### PR TITLE
Addition of ConfidenceInterval class

### PR DESCRIPTION
**Pull request by marinang**

### CODE REVIEW

File: skstats/hypotests/core/confidence_interval.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                >>> from skstats.hypotests.calculators import AsymptoticCalculator
+                >>> from skstats.hypotests import ConfidenceInterval
+                >>> from skstats.hypotests.parameters import POI
+
+                >>> calculator = AsymptoticCalculator(loss, MinuitMinimizer())
+                >>> poinull = POI(mean, np.linspace(1.15, 1.26, 100))
+                >>> ci = ConfidenceInterval(calculator, [poinull])
+                >>> ci.interval()
+                Confidence interval on mean:
+                    1.1810371356602791 < mean < 1.2156701172321935 at 0.7% C.L.
```
Comment (eduardo-rodrigues) [inline skstats/hypotests/core/confidence_interval.py]: At 0.7%? I'm confused ...

File: skstats/hypotests/core/confidence_interval.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        """
+
+        return self.calculator.pvalue(self.poinull, qtilde=self.qtilde, onesided=False)[0]
+
+    def interval(self, alpha=0.32, printlevel=1):
+        """
+        Returns the confidence level on the parameter of interest.
+
+        Args:
+            alpha (float, default=0.05): significance level
```
Comment (eduardo-rodrigues) [inline skstats/hypotests/core/confidence_interval.py]: Default is inconsistent ;-). Worth saying the default is just 1 sigma ...

File: skstats/hypotests/core/confidence_interval.py (hunk 1 above)
Comment (marinang) [reply skstats/hypotests/core/confidence_interval.py]: There was a typo in the output that I fixed, but I didn't copy pasted the new output here. Should be 68 %.

File: skstats/hypotests/core/confidence_interval.py (hunk 2 above)
Comment (marinang) [reply skstats/hypotests/core/confidence_interval.py]: Yes should 0.32. I can say that is it one sigma as well.

**Comment by marinang** [comment]
@eduardo-rodrigues thanks. Actually using the asymptotic calculator the confidence interval computed is a profile likelihood interval, which you can get just by calling Minos with Minuit.

**Pull request merged**